contrib/lib/src/space_helmet/helmet.rs

```diff
@@ -0,0 +1,131 @@
+use std::sync::atomic::{AtomicBool, Ordering};
+
+extern crate rocket;
+use rocket::fairing::{Fairing, Info, Kind};
+use rocket::http::Header;
+use rocket::Rocket;
+use rocket::{Request, Response};
+
+use space_helmet::policy::*;
+
+pub struct Helmet<'a> {
```

Comment:
All public items should be documented with usage examples. See, as an example, the documentation on the [`Template`](https://api.rocket.rs/rocket_contrib/struct.Template.html) type.